src/Doctrine/Instantiator/InstantiatorInterface.php

```diff
@@ -16,5 +16,5 @@
      *
      * @throws ExceptionInterface
      */
-    public function instantiate($className);
+    public function instantiate(string $className);
```

Comment:
This is a major BC break, as it breaks implementations :-\

Reply:
> This is a major BC break, as it breaks implementations

Note: Not with PHP 7.2+.

But then it should have `: object` return type, which does break implementations (but can be made forward compatible with both Instantiator 1.x and 2.x).